runtime/vault-filesystem/src/main/java/io/aklivity/zilla/runtime/vault/filesystem/internal/FileSystemVaultHandler.java

```diff
@@ -66,7 +72,7 @@ public FileSystemVaultHandler(
             : aliases -> null;
 
         FileSystemStoreInfo trust = supplyStoreInfo(resolvePath, options.trust);
-        supplyTrust = (aliases, cacerts) -> newTrustFactory(trust, aliases, cacerts);
+        supplyTrust = (aliases, cacerts, crlChecks) -> newTrustFactory(trust, aliases, cacerts, crlChecks);
```

Comment:
This forced API change suggests that the decision to enforce CRL checks might belong in the vault, not the TLS binding, what do you think?